vue/src/components/NoteFooter.vue

```diff
@@ -1,3 +1,4 @@
+<!-- This footer is displayed beneath every note. The deletion button only is shown if the user is authorized to delete it -->
```

Comment:
"This footer is displayed under the content of every note. The deletion button is only shown if the user is authorized to delete it" or just "This footer is part of every note. The deletion button is only shown if the user is authorized to delete it"